src/main/resources/com/google/jenkins/plugins/computeengine/InstanceConfiguration/help-localSsdDisks.html

```diff
@@ -0,0 +1,10 @@
+<div>
```

Comment:
You might want to remove this file if not including local SSD support.

Reply:
done